smithy-openapi/src/main/java/software/amazon/smithy/openapi/model/ParameterObject.java

```diff
@@ -246,18 +246,18 @@ public Builder schema(Schema schema) {
             return this;
         }
 
-        public Builder example(Node example) {
+        public Builder example(ExampleObject example) {
```

Comment:
This seems like it _could_ break someone. Can we keep a corresponding methods that take `Node` and then use `ExampleObject#fromNode`?

Reply:
ExampleObject doesn't have a `fromNode` method, but sure - it can be made.

Reply:
resolved.